.changeset/olive-apples-brush.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/transactions": patch
```

Comment:
This will be a breaking change for `@fuel-ts/transactions`, people _probably_ aren't using these but we probably need to deprecate first then remove them in the major release.

Reply:
We [export](https://github.com/FuelLabs/fuels-ts/pull/3106/files#diff-899d0327255b7d5fc6591aba06bca1755fddc31c2956a960b2328be955115d3eL4) these coders, therefore, I'd favour a deprecation notice for now.

Reply:
The only plausible reason someone would use these coders is to access the static methods that some of them provide, such as:

-  [ReceiptMintCoder.getAssetId](https://github.com/FuelLabs/fuels-ts/blob/2caf1b1d4d2f05d47eb468ab8481e53dd07dfd5c/packages/transactions/src/coders/receipt.ts#L787)
- [ReceiptBurnCoder.getAssetId](https://github.com/FuelLabs/fuels-ts/blob/2caf1b1d4d2f05d47eb468ab8481e53dd07dfd5c/packages/transactions/src/coders/receipt.ts#L855)
- [ReceiptMessageOutCoder.getMessageId](https://github.com/FuelLabs/fuels-ts/blob/2caf1b1d4d2f05d47eb468ab8481e53dd07dfd5c/packages/transactions/src/coders/receipt.ts#L692)

Beyond these methods, there's no real reason to use the receipt coders. Currently, none of the GraphQL responses include serialized receipts.

Additionally, I highly doubt that all of these coders are working as expected. We stopped using them over a year ago, so it's very likely they need adjustments to work properly now.

I can either:

1. Maintain and deprecate all of them.
2. Remove most of them and maintain and deprecate only the ones with static methods. 

Let me know what you guys think.

Reply:
The safer option is to maintain and deprecate all of them. Given there are very few use cases for them, it is probably not that much of a maintenance load and so the cons outweigh the risk of removing them altogether. They can be removed at our next breaking release cycle.